src/DssSpell.t.sol

```diff
@@ -1230,6 +1074,22 @@ contract DssSpellTest is DSTest, DSMath {
         }
         assertEq(sumlines, vat.Line());
     }
+    
+    function getOSMPrice(address pip) internal returns (uint256) {
+        // hevm.load is to pull the price from the LP Oracle storage bypassing the whitelist
+        uint256 price = uint256(hevm.load(
+            pip,
+            bytes32(uint256(3))
+        )) & 0xFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF;
```

Comment:
Maybe ` & uint128(-1)` for easier auditing and a comment here that you need to shift off the back half of the packed storage value.